ionic_user_interface/src/views/About.vue

```diff
@@ -4,19 +4,34 @@
       <div id="container">
         <strong>Hello Climbers! &#129495;</strong>
         <p class="lead">
-          Personally, when people make routes for me at the gym,
-          <br />
-          I tend to forget half of the moves due to my poor memory.
+          This is a free open source project between two friends who happen to code and climb.
         </p>
 
         <p class="lead">
-          One way is to draw them out on a handphone,
+          It aims to accomplish two things:
           <br />
-          but it is difficult to do so with fat fingers, and labeling
+        </p>
+        <p class="lead">
+          1. Provide a drawing board for custom climbing routes
           <br />
-          the exact order of moves can be troublesome too.
+          2. Route sharing system where climbers can search routes by gym, grade and vote
         </p>
-        <p class="lead">This application aims to solve this issue.</p>
+
+        <p class="lead">Please star our repository!</p>
+
+        <ion-button
+          target="_blank"
+          href="https://github.com/nandium/RouteMaker"
+          color="light"
+          size="large"
+        >
+          <ion-icon class="github-logo" :icon="logoGithub" size="medium"></ion-icon>
+          Github
+        </ion-button>
+
+        <br />
+        <br />
+        <strong>How To Use</strong>
```

Comment:
The spacing below this title is larger than spacing above this title, please flip it around